### DISCUSSION TITLE
nips-ja ユビキタス言語 Wiki 追加

### DISCUSSION THREAD

**Discussion by Hakkadaikon** [nostr-jp/nips-ja#78] [Announcements]
nips-jaのリポジトリに、ユビキタス言語のWikiを追加しました。
仮運用として、統一させたい文言がある場合、このWikiに追記してもらえばと思います。
https://github.com/nostr-jp/nips-ja/wiki/NIPs-ユビキタス言語

**Comment by penpenpng** [comment]
原文の方を辞書順に並べると重複を回避できるし、対訳を探しやすいので良さそうです。

**Reply by Hakkadaikon** [reply]
今は３つしか書いてなかったので、手動で辞書順 (英)に並べ替えました。
markdown形式テーブルのソートや整形に便利なツール・プラグインなどもしご存じでしたら
どなたか教えていただけると助かります。

**Reply by mattn** [reply]
vim で CSV 形式のテキストを Markdown の表にしたいのであれば

https://github.com/mattn/vim-maketable

![](https://camo.githubusercontent.com/d7356315137bf0088c584a94e7c6adf20e28090d63782c8048f94b290997cb34/687474703a2f2f692e696d6775722e636f6d2f6a6f59475665322e676966)

**Reply by jiftechnify** [reply]
VSCodeの場合、Makrdown Tableという拡張機能を使うと、`tab`キーで次の欄に移動しつつ自動的に整形してくれるので便利です。
https://marketplace.visualstudio.com/items?itemName=TakumiI.markdowntable

**Comment by s3-odara** [comment]
markdownの表のインデントって手動でスペース入れるパワープレイですか？
それとも何か方法があります？

**Reply by mattn** [reply]
表のインデントってどんなのを想像されておられます？

**Reply by s3-odara** [reply]
すみません。メンション気づきませんでした。
wikiってwebからしか編集できないかと思っていたんですがローカルで編集できそうなので適当なエディタで編集します。

**Comment by Hakkadaikon** [comment]
wikiを作成したので、以下のユビキタス言語策定のissueはクローズさせます。
https://github.com/nostr-jp/nips-ja/issues/77

**Reply by Hakkadaikon** [reply]
クローズさせました。
